test/e2e-common.sh

```diff
@@ -27,7 +27,7 @@ source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
 function install_pipeline_crd() {
   echo ">> Deploying Tekton Pipelines"
-  kubectl apply -f https://storage.googleapis.com/tekton-releases/latest/release.yaml || fail_test "Tekton Pipeline installation failed"
+  kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml || fail_test "Tekton Pipeline installation failed"
```

Comment:
It's better to use what we are doing in CLI. Getting it from Github release. https://github.com/tektoncd/cli/blob/master/test/e2e-common.sh#L129